src/components/DigitalCredentials/CredentialsTable.vue

```diff
@@ -10,72 +10,68 @@
         fixed
         disable-sort
         hide-default-footer
-        :items="credentialRecords"
-        :headers="!!credentialRecords ? credentialsTableHeaders : []"
+        :items="issuedCredentials"
+        :headers="!!issuedCredentials ? credentialsTableHeaders : []"
```

Comment:
I don't think you can check for empty arrays like this. You may have to check length instead.

Please test and lmk what you intend here and if it works as you expect.

Reply:
removed! As you noted below, was already doing a check for this :)